src/main/java/com/labsynch/labseer/web/AnalysisGroupController_Roo_Controller.aj

```diff
@@ -113,9 +113,7 @@ privileged aspect AnalysisGroupController_Roo_Controller {
         if (enc == null) {
             enc = WebUtils.DEFAULT_CHARACTER_ENCODING;
         }
-        try {
-            pathSegment = UriUtils.encodePathSegment(pathSegment, enc);
-        } catch (UnsupportedEncodingException uee) {}
+        pathSegment = UriUtils.encodePathSegment(pathSegment, enc);
```

Comment:
That's a major change. Did you try getting Roo to re-generate this .aj file? Wondering if it would produce something that compiles if prompted to regenerate the aj file or whether it is just generating broken code now.

Reply:
We removed ROO to resolve this.